src/runner/main.cpp

```diff
@@ -23,13 +23,78 @@
 
 #include <winrt/Windows.System.h>
 
+#include <Psapi.h>
+#include <RestartManager.h>
+
 #if _DEBUG && _WIN64
 #include "unhandled_exception_handler.h"
 #endif
 #include <common/notifications/fancyzones_notifications.h>
 
 extern "C" IMAGE_DOS_HEADER __ImageBase;
 
+const wchar_t ExplorerProcessName[] = L"explorer.exe";
+const wchar_t PTUpdateMessageBoxTitle[] = L"PowerToys";
+const wchar_t PTUpdateMessageBoxText[] = L"PowerToys was updated and some components require Windows Explorer to restart. Do you want to restart Windows Explorer now?";

```

Comment:
Let's move it into `localized_strings` namespace in this file

Reply:
Done.